config/settings.yml

```diff
@@ -1480,6 +1480,6 @@
       :memory_threshold: 1.gigabytes
       :nice_delta: 1
     :websocket_worker:
-       :connection_pool_size: 8
+       :connection_pool_size: 26
```

Comment:
@jrafanie seems like the logger can't deal with the DB connections initiated from ActionCable because it's not changing the numbers at all :disappointed: